packages/manager/modules/pci/src/projects/new/routing.js

```diff
@@ -17,71 +17,94 @@ export default /* @ngInject */ ($stateProvider) => {
-      const windowPromise = transition.injector().getAsync('$window');
-      const coreURLBuilderPromise = transition
-        .injector()
-        .getAsync('coreURLBuilder');
-      const cartPromise = transition.injector().getAsync('cart');
-      const eligibilityPromise = transition.injector().getAsync('eligibility');
-      const newSupportTicketLink = transition
-        .injector()
-        .getAsync('newSupportTicketLink');
+      const injector = transition.injector();
+
+      const translatePromise = injector.getAsync('$translate');
+      const windowPromise = injector.getAsync('$window');
+      const coreURLBuilderPromise = injector.getAsync('coreURLBuilder');
+      const cartPromise = injector.getAsync('cart');
+      const eligibilityPromise = injector.getAsync('eligibility');
+      const newSupportTicketLink = injector.getAsync('newSupportTicketLink');
+      const isTrustedZonePromise = injector.getAsync('isTrustedZone');
+      const projectsPromise = injector.getAsync('projects');
+
       return Promise.all([
         translatePromise,
         windowPromise,
         cartPromise,
         eligibilityPromise,
         coreURLBuilderPromise,
-      ]).then(([$translate, $window, cart, eligibility, coreURLBuilder]) => {
-        let redirectState = 'pci.projects.new.config';
-        let redirectParams = transition.params();
-        const redirectOptions = {
-          location: false,
-        };
-
-        if (eligibility.isAskIncreaseProjectsQuotaRequired()) {
-          redirectState = 'pci.projects.project.error';
-          redirectParams = {
-            message: $translate.instant(
-              'pci_project_new_error_ask_increase_projects_quota',
-            ),
-            code: ELIGIBILITY_ACTION_ENUM.ASK_INCREASE_PROJECTS_QUOTA,
-            image: ELIGIBILITY_ERROR_IMAGES_SRC.ASK_INCREASE_PROJECTS_QUOTA,
-            projectId: get(transition.params('from'), 'projectId'),
-            submitLabel: $translate.instant(
-              'pci_project_new_error_contact_support',
-            ),
-            submitLink: newSupportTicketLink,
+        isTrustedZonePromise,
+        projectsPromise,
+      ]).then(
+        ([
+          $translate,
+          $window,
+          cart,
+          eligibility,
+          coreURLBuilder,
+          isTrustedZone,
+          projects,
+        ]) => {
+          let redirectState = 'pci.projects.new.config';
+          let redirectParams = transition.params();
+          const redirectOptions = {
+            location: false,
           };
-        } else if (eligibility.isVerifyPaypalRequired()) {
-          redirectState = 'pci.error';
-          redirectParams = {
-            message: $translate.instant('pci_project_new_error_verify_paypal', {
-              href: coreURLBuilder.buildURL(
-                'dedicated',
-                '#/billing/payment/method',
+
+          if (isTrustedZone) {
+            return {
```

Comment:
why redirect back to onboarding page for trusted nic? do we show a message like "contact support team to create new project"?

Reply:
Yes, customers have to contact support to create a project.
About redirect to onboarding page, true we have to write a message to customers to know why they can't create a project.

I already highlight this point to PU, but at the moment no response.
I think it will be certainly highlighted also by QC to add a message.